Content.Server/PowerSink/PowerSinkSystem.cs

```diff
@@ -32,24 +52,82 @@ private void OnExamine(EntityUid uid, PowerSinkComponent component, ExaminedEven
 
         public override void Update(float frameTime)
         {
-            var toRemove = new RemQueue<(PowerSinkComponent Sink, BatteryComponent Battery)>();
+            var toRemove = new RemQueue<(EntityUid Entity, PowerSinkComponent Sink, BatteryComponent Battery)>();
+            var query = AllEntityQuery<PowerSinkComponent, PowerConsumerComponent, BatteryComponent, TransformComponent>();
```

Comment:
```suggestion
            var query = EntityQueryEnumerator<PowerSinkComponent, PowerConsumerComponent, BatteryComponent, TransformComponent>();
```

AllEntityQuery also includes paused entities.